build_projects/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/BuildPlatformsAttribute.cs

```diff
@@ -18,11 +19,17 @@ public BuildPlatformsAttribute(params BuildPlatform[] platforms)
             _buildPlatforms = platforms;
         }
 
+        public BuildPlatformsAttribute(BuildPlatform platform, string version)
+        {
+            _buildPlatforms = new BuildPlatform[] { platform };
+            _version = version;
+        }
+
         public override bool EvaluateCondition()
         {
             foreach (var platform in _buildPlatforms)
             {
-                if (CurrentPlatform.IsPlatform(platform))
+                if (CurrentPlatform.IsPlatform(platform, _version))
```

Comment:
Can we add additional build platforms that correspond to specific versions rather than do this? With this there's no way I could say for example, only run this target on ubuntu 14.04, debian 8.1.

The build platforms were never intended to specify groups of builds on which the targets should be run on, but rather to target specific build definitions.